src/Helper/Styles.php

```diff
@@ -85,7 +83,7 @@ public function addCond($cond, $href, array $attr = null, $pos = 100)
      *
      * @access protected
      */
-    protected function style($css, array $attr = null)
+    protected function style($css, $attr = null)
```

Comment:
Same issue with type hint and null. My comments regarding 

`array $variable = null` to `array $variable = []` stay same for all the below methods also.

Reply:
@harikt It was a hack to prevent deprecated errors in PHP 8.4 while keeping older PHP versions intact, and it had remained. I fixed it.